src/styles/sidebar/components/NotebookView.scss

```diff
@@ -29,6 +29,11 @@
 
   &__items {
     grid-area: items;
+    // Setting a min-width on this grid area is necessary to keep long strings
+    // in child elements from busting out; it's needed to make
+    // `overflow-wrap: break-word` work in descendants. See
+    // https://css-tricks.com/preventing-a-grid-blowout/
+    min-width: 0;
```

Comment:
There is already a reference to this article above in the `grid-template-columns` rule for `.NotebookView`. That defines a minimum size of 0 for the track. However, when the `responsive.tablet-and-up` mode defined below is active, there are two columns. The `grid-template-columns` rule above only defines a rule for one column. My guess is that the second column ends up with a default sizing policy. I tried adding `grid-template-columns: minmax(0, auto) minmax(0, auto)` to the `resolve.tablet-and-up` block as an alternative to the fix here and that resolved the issue as well.